lib/MusicBrainz/Server/Validation.pm

```diff
@@ -98,6 +102,12 @@ sub is_database_bigint_id {
     is_positive_integer($t) and $t <= $MAX_POSTGRES_BIGINT;
 }
 
+sub has_oneline_string_length {
```

Comment:
This name is confusing to me - seeing it I wouldn't know if it means it's over the limit or under the limit. What about something boring like `is_not_too_long_string`?

Reply:
It would be less confusing w.r.t. “over/under” but it doesn’t say for what it “is not too long”.

Would `is_short_oneline_string` carry both meanings?

Reply:
`is_short_enough_oneline_string`? `is_short_enough_single_line_string`? Just `is_short` seems a bit funny because 1024 characters is not short :D But I can live with that too really.

Reply:
I renamed it `has_at_most_oneline_string_length` in https://github.com/metabrainz/musicbrainz-server/pull/3251/commits/4ee30f845aa06df6482640b4c92fa0743aca2641 to keep it meaningful and not confusing.